src/organisms/forms/GeneralForm/Readme.md

```diff
@@ -7,22 +7,26 @@ _This is a form wrapper that can take components as children._
       onSubmit={ () => alert('FormSubmitted!') }
     >
       <TextComponent tag='p' type={ 3 } regular>First, a little about you and your humble abode</TextComponent>
+
       <TextField label='First name' placeholder='Enter your first name here' forProp=''/>
+
       <LinkWrapper href="#">
         <TextComponent>
           Want to add more people to the policy?
         </TextComponent>
       </LinkWrapper>
-      <SelectField
-        forProp=''
-        label='Home type'
-        placeholder='Select your home type'
+
+      <SimpleSelectField
+        label='Select Field Label'
+        placeholder='Placeholder'
         selectOptions={[
-        ['Value 1', 'Option 1'],
-        ['Value 2', 'Option 2'],
-        ['Value 3', 'Option 3']
+          { value: 'Option 1' },
+          { value: 'Option 2' },
+          { value: 'Option 3' }
```

Comment:
@jmcolella Alright. But I feel like this is the same issue we discussed before about not giving people the option to create the mappings. What if the value needs to correspond to some other data object, but we want to display a pretty name for the user on the FE?

Also, this still doesn't guarantee order -- so, if we wanted a list of months in a certain order or like a list of years that are 100+ options long?